lib/setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from distutils.core import setup
+from pathlib import Path
+
+package_name = 'wit'
+
+assets = [
+    "scala-bridge-fetcher_2.12-0.1.0.jar"
+]
+
+scripts = [
+    'bin/wit',
+]
+
+setup(
+    name=package_name,
+    version='0.1',
```

Comment:
Let's make this match the actual version number of the tool. At the moment, we are at 0.12.0.